src/app/contacts/services/contact.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { Contact } from '../models/contactDTO';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { BehaviorSubject } from 'rxjs';
+import { environment } from '../../../environments/environment';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class ContactService {
+  contacts: Contact[];
+  contactsSource = new BehaviorSubject([]);
+
+  constructor(private http: HttpClient) {
+    this.getContacts()
+  }
+
+  getContacts = () => {
+    let headers = new HttpHeaders();
+    headers = headers.append('Authorization', 'Basic ' + btoa('MY_SECRET_ENCRYPTED_TOKEN'));
```

Comment:
These headers are not necessarily here because they will be overridden in the http_interceptor

Reply:
I write it here in the first, then transfer it to Interceptor and forget to remove from here.